ds_config/other_zero3.json

```diff
@@ -0,0 +1,50 @@
+{
+    "fp16": {
+        "enabled": "auto",
+        "loss_scale": 0,
+        "loss_scale_window": 1000,
+        "initial_scale_power": 16,
+        "hysteresis": 2,
+        "min_loss_scale": 1
+    },
+    "bf16": {
+        "enabled": "auto"
+    },
+    "optimizer": {
+        "type": "AdamW",
```

Comment:
We can use paged adam if we need better memory consumption

Reply:
Not sure if paged adam works with deepspeed, I will test and confirm